xml_analyzer/formula.py

```diff
@@ -1,13 +1,38 @@
 from Equation import Expression
-import numpy
+from .definitions import EquationTypes
 
 
 class Formula:
     def __init__(self, constraint_spec: str = "", variables: list = []):
-        self.expression = expression
-        self.variables = variables
+        self.expression = self._make_expression(constraint_spec, variables)
 
-    def make_expression(self, expression: str):
-        return Expression(self.expression, self.variables)
+    def _make_expression(self, expression: str, variables: list):
+        expr_type, expr_sep = self._get_expression_type(expression)
+        if expr_type == EquationTypes.equation:
+            expression = expression.split(expr_sep)[1]
 
+        return Expression(expression, variables)
+
+    @staticmethod
+    def _get_expression_type(expr: str) -> [EquationTypes, str]:
```

Comment:
expression is not much longer than expr. Especially not longer than ex STRG+LEERTASTE ENTER

Reply:
Fixed with https://github.com/Squiddrone/VPParametricsSolver/pull/2/commits/94c7a5274c3d122e82bf28a3421db92e9d3cfdd1